demos/multichannel_demo/hpe/README.md

```diff
@@ -101,7 +101,11 @@ Alternatively, you can just set `-nc 3`, which simplifies application usage.
 
 If your cameras are connected to PC with indexes gap (for example, `0,1,3`), use the `-i` parameter.
 
-IP-cameras through RSTP URI interface are not supported.
+Added support for IP-cameras through RSTP URI interface.
```

Comment:
A README should document the current state, not changes. E.g. you could say "IP cameras are supported via the RTSP URI interface:".